image-patching/stopgap-script/README.md

```diff
@@ -14,6 +14,8 @@ Copy the overcloud-full.qcow2 from undercloud-director /home/stack/images/ to th
 
 `cp overcloud-full.qcow2 overcloud-full-bk.qcow2`
 
+Untar the os-net-config.tar.gz provided by Mellanox and place it in the same location as overcloud-full.qcow2.
```

Comment:
This is just WA until all the patches of os-net-config will be merged and available by redhat. Do we need to document this here?

Reply:
Hi Moshe, I have updated this document with with "Note" https://github.com/nuagenetworks/nuage-ospdirector/pull/54/files#diff-2bec545a520361c8df6f55f470ed44b6R17. Please let me know how it looks. I am open to suggestions.